README.md

```diff
@@ -98,6 +101,33 @@ router
 
 ### Generating Type-Safe Clients
 
+#### From the Same Project
```

Comment:
This `OneSchemaRouter.client` functionality already existed, but I'm documenting it here because its a very useful feature.